addons/hr_holidays/models/hr_leave_allocation.py

```diff
@@ -175,14 +176,32 @@ def _compute_description_validity(self):
                 name_validity = _(
                     "%(allocation_name)s (from %(date_from)s to %(date_to)s)",
                     allocation_name=allocation.name,
-                    date_from=allocation.date_from.strftime("%b %d %Y"),
-                    date_to=allocation.date_to.strftime("%b %d %Y"),
+                    date_from=format_date(
+                        allocation.env,
+                        fields.Date.context_today(
+                        allocation,
+                        fields.Datetime.to_datetime(allocation.date_from),
+                        ),
+                    ),
```

Comment:
I think this would be better for readability 
```suggestion
                    date_from=format_date(allocation.env,
                        fields.Date.context_today(allocation, fields.Datetime.to_datetime(allocation.date_from)),
                    ),
```